docs/concept/TAP7.1-solution-design.md

```diff
@@ -0,0 +1,425 @@
+
+## 4.2 Process
+
+### 4.2.1 Use Case View (high-level)
+
+| Use Case                                | Description                                                                                                       | Actors            |
+|-----------------------------------------|-------------------------------------------------------------------------------------------------------------------|-------------------|
+| Configure HELM Chart                    | The product developer configures the HELM Chart for the specific environment.                                     | Product Developer |
+| Configure Secrets                       | The product developer manages and configures secrets.                                                             | Product Developer |
+| Execute Deployment Pipeline             | The product developer successfully executes the deployment pipeline for the HELM Chart.                           | Product Developer |
+| Integrate EDC, DTR, and Identity Wallet | The actors integrate the components EDC, Digital Twin Registry, and Identity Wallet for seamless interoperability | Product Developer |
+| Set Up Connection Infrastructure        | The actor sets up the connection infrastructure for linking to the central data space infrastructure.             | Product Developer |
+| Maintain Infrastructure Module          | The product developer maintains the modular architecture of the HELM Chart.                                       | Product Developer |
+| Ensure Scalability                      | The product developer ensures scalability to support growing numbers of users and data volumes.                   | Product Developer |
+| Sandbox Environments                    | The product developer creates local environments to evaluate and experiment with Catena-X components.             | Product Developer |
+| E2E Testing                             | The product tester runs end-to-end tests for Catena-X services.                                                   | Product Tester    |
+
+### 4.2.2 Business Process (SCC Artifact)
+
+#### Chart Release Process
+
+Charts will be released using the already existing workflow in the Umbrella Chart repository. The workflow will
+automatically release a Chart, once it is pushed to the "charts/" directory with a new Chart version.
```

Comment:
Any more detail info how this is versioned or why.

Reply:
added a remark to semantic versioning